app/components/SubscriptionsBanner.tsx

```diff
@@ -26,7 +26,7 @@ export const SubscriptionsBanner = () => {
     fetchSubscription().then(({subscriptions}) => {
       setShowBanner(subscriptions.length === 0);
     });
-  }, []);
+  });
```

Comment:
Won't this happen on every component update now?

Reply:
Ya. I started noticing now with the dedupped retrieve calls in `app/api/account_session/route.ts`

That the subscriptions was getting stale. Ima followup and cache the subscriptions api. Though I tested an it really isn't that bad since the only component updates are switching pages.